framework/include/interfaces/TaggingInterface.h

```diff
@@ -130,6 +130,9 @@ class TaggingInterface
   /// The residual tag ids this Kernel will contribute to
   std::set<TagID> _vector_tags;
 
+  /// The reference residual tag ids
```

Comment:
Like @loganharbour said, I don't think we have to tie this absolute value business to `ReferenceResidualProblem`. I'd like to see all language related to reference removed from this class